jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoostModel.scala

```diff
@@ -42,6 +42,21 @@ abstract class XGBoostModel(protected var _booster: Booster)
   extends PredictionModel[MLVector, XGBoostModel] with BoosterParams with Serializable
     with Params with MLWritable {
 
+  private var _trainingSummary: Option[XGBoostTrainingSummary] = None
```

Comment:
why underscore?

Reply:
Removed.

Reply:
Actually, it was mimicking the naming in `XGBoostModel`, but I have no strong preferences on this.